kubernetes/client/clientset/typed/compose/v1beta2/compose_client.go

```diff
@@ -0,0 +1,74 @@
+package v1beta2
+
+import (
+	"github.com/docker/cli/kubernetes/client/clientset/scheme"
+	v1beta2 "github.com/docker/cli/kubernetes/compose/v1beta2"
+	serializer "k8s.io/apimachinery/pkg/runtime/serializer"
+	rest "k8s.io/client-go/rest"
+)
+
+// ComposeV1beta2Interface defines the methods a compose v1beta2 client has
+type ComposeV1beta2Interface interface {
+	RESTClient() rest.Interface
+	StacksGetter
+}
+
+// ComposeV1beta2Client is used to interact with features provided by the compose.docker.com group.
+type ComposeV1beta2Client struct {
+	restClient rest.Interface
+}
+
+// Stacks returns a stack client
+func (c *ComposeV1beta2Client) Stacks(namespace string) StackInterface {
+	return newStacks(c, namespace)
+}
+
+// NewForConfig creates a new ComposeV1beta2Client for the given config.
+func NewForConfig(c *rest.Config) (*ComposeV1beta2Client, error) {
+	config := *c
+	if err := setConfigDefaults(&config); err != nil {
+		return nil, err
+	}
+	client, err := rest.RESTClientFor(&config)
+	if err != nil {
+		return nil, err
+	}
+	return &ComposeV1beta2Client{client}, nil
+}
+
+// NewForConfigOrDie creates a new ComposeV1beta2Client for the given config and
+// panics if there is an error in the config.
+func NewForConfigOrDie(c *rest.Config) *ComposeV1beta2Client {
```

Comment:
`NewForConfigOrDie()` is only used by the wrapper `NewForConfigOrDie()`, which doesn't seem to be used?

Reply:
Same as above.